source/operator.rst

```diff
@@ -1241,6 +1288,26 @@ options for Percona Server for MondoDB :ref:`sharding<operator.sharding>`.
 +-----------------+---------------------------------------------------------------------------------------------+
 |														|
 +-----------------+---------------------------------------------------------------------------------------------+
+|                 | .. _sharding-mongos-configuration:								|
+|                 |												|
+| **Key**         | `sharding.mongos.configuration <operator.html#sharding-mongos-configuration>`_		|
++-----------------+---------------------------------------------------------------------------------------------+
+| **Value Type**  | string											|
++-----------------+---------------------------------------------------------------------------------------------+
+| **Example**     | .. code:: yaml										|
+|                 |												|
+|                 |     |											|
+|                 |     operationProfiling:									|
+|                 |       mode: slowOp										|
```

Comment:
`mongos` doesn't support `operationProfiling.mode` option so I propose to remove it and just leave systemLog example.